docs/en/contributing/install.md

```diff
@@ -8,33 +8,33 @@ sidebar_label: Install
 
 ## Prerequisites
 
-Before setting up FAST-DNA, install Git and [yarn](https://yarnpkg.com/en/docs/install).
+Before setting up FAST-DNA, install Git and <a href="https://yarnpkg.com/en/docs/install" target="_blank">Yarn</a>.
 
 ## Setup the source repository
 
-Clone the repository, `cd` into the project, install [Lerna](https://github.com/lerna/lerna) and install dependencies:
+Clone the repository, `cd` into the project, install <a href="https://github.com/lerna/lerna" target="_blank">Lerna</a> and install dependencies:
 
 ```shell
 git clone https://github.com/Microsoft/fast-dna.git
 cd fast-dna
 yarn global add lerna
 ```
 
-Install all dependencies:
+Installs all the dependencies:
```

Comment:
Can you elaborate on the need for this change? This now reads as a description of what the command does rather than the declaration of what to do / how to setup the source repo. If the change here is meant simply as documentation of setting up the source repo, I would expect this to follow something along the lines of line 35.

Reply:
If the command says what it does, then why have a label at all?   I was attempting to create a more consistent, meaningful, and readable label using proper grammar.  You're free to suggest something else, but, I'm working to improve our documentation where I see opportunity as I run into things.

Reply:
This seems to more closely match the pattern like below where the testing is outlined - essentially, "here are all the commands". I think the initial intent was how to get the packages up and running. I'm fine with either, but having these in their own blocks seems inconsistent now.

Reply:
Can we just remove all the commands completely, after all, they everyone knows they are in the package.json

Reply:
I'm fine with that, but not going to block on this. I think this can go in as it is. The update to the formatting just confused me a bit.